tests/unit/drivers/sql/test_amazon_redshift_sql_driver.py

```diff
@@ -162,4 +162,4 @@ def test_execute_query(self, statement_driver):
         assert statement_driver.execute_query("query") == [BaseSqlDriver.RowResult(row) for row in rows]
 
     def test_get_table_schema(self, describe_table_driver):
-        assert describe_table_driver.get_table_schema("dev") == ["first_name", "last_name"]
+        assert describe_table_driver.get_table_schema("dev") == "['first_name', 'last_name']"
```

Comment:
This doesn't feel quite right, `get_table_schema` should still return a list.

Reply:
Change was made to match the changes in `amazon_redshift_sql_driver`